integration/swarm/swarm.bats

```diff
@@ -66,6 +70,29 @@ setup() {
 	done
 }
 
+@test "obtain hostname of the replicas" {
+	# this will help to obtain the hostname of
+	# the replicas from the curl
+	unset http_proxy
+	check_status=$($DOCKER_EXE ps --filter status=running --format "{{.ID}}" | wc -l)
+	if [ "$check_status" -lt "$number_of_replicas" ]; then
+		sleep 1
```

Comment:
This isn't going to be reliable. I think you'll have to repeatedly check for up to, say, 60 seconds and if the number isn't correct, then fail.